src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -1681,17 +1697,69 @@ def check_memory_size
       return if !Mode.installation
       # less than LOW_MEMORY_MIB RAM, the 64MiB buffer is for possible
       # rounding in hwinfo memory detection (bsc#1045915)
```

Comment:
Please move this comment also below, the "64MiB buffer" part belongs to that `- 64` subtraction.

Reply:
Ah, right. Moved.